sklearn/metrics/_regression.py

```diff
@@ -1499,7 +1499,17 @@ def d2_pinball_score(
 
     return np.average(output_scores, weights=avg_weights)
 
-
+@validate_params(
```

Comment:
yes absolutely! doing it right now.
I apologize for the stupid question(First time contributing), simply add it and push it on the same branch after doing the pytest?

Reply:
You have to make the change on your local branch and execute `pytest` locally for this `test_public_functions.py` file. If it passes, then you can safely commit and push on your remote. This will synchronize this pull-request and our continuous integration will run your changes on all supported platforms.

Reply:
That's exactly what I did this afternoon.